### PR TITLE
Fix rahash2 entropy to show non truncated double

### PR DESCRIPTION
**Pull request by faridtsl**
Past behaviour : 
entropy : 04000000
Current behaviour :
entropy : 4.39653582

### CODE REVIEW

**Comment by radare** [comment]
can you update the tests? https://github.com/radare/radare2-regressions/pulls

**Comment by radare** [comment]
Seems like this patch is making some ansi escape codes to appear in the output and breaks many tests

File: binr/rahash2/rahash2.c [hunk 1]
```diff
@@ -72,17 +72,29 @@ static void do_hash_seed(const char *seed) {
...
-		for (i = len - 1; i >= 0; i--) {
-			printf ("%02x", c[i]);
+	if (!len) {
+		double e = r_hash_entropy (buf, len);
+		if (rad) {
+			printf ("entropy: %.8f\n", e);
+		} else {
+			printf ("0x%08"PFMT64x "-0x%08"PFMT64x " %.8f: ",
+				from, to > 0? to - 1: 0, e);
+			r_print_progressbar (NULL, 12.5 * e, 60);
```
Comment (radare) [inline binr/rahash2/rahash2.c]: probably this is the reason of travis being red

**Comment by radare** [comment]
also please rebase on top of master

**Comment by radare** [comment]
Please squash and rebase. Dont merge in prs ever. Also update the tests in another pr for r2r

File: binr/rahash2/rahash2.c (hunk 1 above)
Comment (faridtsl) [reply binr/rahash2/rahash2.c]: I have a PR on r2r adding a new test for entropy, other tests pass locally.
https://github.com/radare/radare2-regressions/pull/1227

**Comment by radare** [comment]
Uhm, in theory if you set the branch of r2r with the same name as the one in here travis should pick your branc and test it, so it should be green

**Comment by radare** [comment]
Thanks!

**Pull request merged**